src/components/Login/Styles.js

```diff
@@ -13,9 +13,16 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
     alignItems: 'center',
   },
+  mainLogoText: {
+    fontFamily: 'ProductSans-Bold',
```

Comment:
I think that we need ttf file also for Xcode, 
and we also add splash to xcode, 
I add after this PR passed